scripts/nxtfilemgr

```diff
@@ -284,12 +285,11 @@ class NXT_FileMgr(Gtk.Window):
             if prot_version == PROTOCOL_VER:
                 brick_name_b, brick_hostid, brick_signal_strength, brick_user_flash = self.brick.get_device_info()
                 connection_type = str(self.brick.sock)
-                brick_name_str = brick_name_b.decode('windows-1252')
-                print("brick_name_b = {0}, brick_name_str = {1}".format(brick_name_b, brick_name_str))
-
+                brick_name_str = brick_name_b.decode('windows-1252').strip('\00')
```

Comment:
I think that `brick.get_device_info()` should return the decoded and stripped value rather than all callers having to do this themselves.

As I recall, the last time I was testing this, it was in fact doing exactly that (tested on Linux and Mac), so I'm not sure why this change would be needed. (I guess I'm going to have to just try it again if I really what to know).

Reply:
Hmm. I'm using nxt-python 2.2.2 which is not the latest master version.
Maybe that is the problem I'm facing, and we need to have a new release version to test with.

Reply:
use the `PYTHONPATH` environment variable

Reply:
> we need to have a new release version to test with.

Yes, [I made this request recently as well](https://github.com/Eelviny/nxt-python/issues/50#issuecomment-385192950)

Reply:
according to the readme of this project, no, it will not be python2 compatible

Reply:
I see. So in that case, I'll still need to maintain my own version of `nxtfilemgr` for 2.2.2.